command.go

```diff
@@ -783,7 +782,7 @@ func (c *Command) initHelpCmd() {
 			Run: func(c *Command, args []string) {
 				cmd, _, e := c.Root().Find(args)
 				if cmd == nil || e != nil {
-					c.Printf("Unknown help topic %#q.", args)
+					c.Printf("Unknown help topic %q\n", args)
```

Comment:
Why the change from `%#q` to   `%q` ? Not that I have a strong opion about `"` quoting vs `` ` `` quoting. But it seems like a needless change...